secant/Dependencies/FeatureFlagsManager/UserDefaultsFeatureFlagsStorage.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  UserDefaultsFeatureFlagsStorage.swift
+//  secant
+//
+//  Created by Michal Fousek on 23.02.2023.
+//
+
+import Foundation
+
+typealias UserDefaultsFeatureFlagsConfigurationProvider = UserDefaultsFeatureFlagsStorage
+typealias UserDefaultsFeatureFlagsManagerCache = UserDefaultsFeatureFlagsStorage
```

Comment:
```suggestion
typealias UserDefaultsConfigurationManagerCache = UserDefaultsFeatureFlagsStorage
```

Reply:
I renamed this to `WalletConfigProviderCache`.